multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/Messages.java

```diff
@@ -190,6 +190,7 @@
     public static final String WILL_NOT_UPDATE_SERVICE_PARAMS = "Service parameters for service \"{0}\" will not be updated, as the option for skipping service parameter updates is enabled.";
     public static final String WILL_NOT_UPDATE_SERVICE_PLAN = "Service plan for service \"{0}\" will not be updated, as the option for skipping service plan updates is enabled.";
     public static final String WILL_NOT_UPDATE_SERVICE_TAGS = "Service tags for service \"{0}\" will not be updated, as the option for skipping service tags updates is enabled.";
+    public static final String WILL_NOT_UPDATE_SYSLOG_URL = "Service syslog drain url for service \"{0}\" will not be updated, as the option for skipping service syslog drain url updates is enabled";
```

Comment:
"syslog drain url" is an implementation-specific name. It will probably not be understandable to the users. Reword the message to make it more user-friendly.